DCode.Web/Views/Requestor/Dashboard.cshtml

```diff
@@ -33,7 +33,7 @@
                 </div>
             </div>
             <div class="input-group new-search-by-project">
-                <input type="text" class="form-control" placeholder="Search by Project Name" ng-model="searchBox.text" ng-change="filterTaskApplicants();">
+                <input type="text" class="form-control" placeholder="Search by Project Name" id="projectNam" ng-model="searchBox.text" ng-change="filterTaskApplicants();">

```

Comment:
Can directly put the text on the placeholder instead of using jquery to inject the same.